packages/amplify-category-function/src/provider-utils/awscloudformation/utils/functionPluginLoader.ts

```diff
@@ -165,6 +167,8 @@ async function getSelectionsFromContributors<T>(
     }
     context.print.info(singleOptionMsg);
     selection = selections[0].value;
+  } else if(isDefaultDefined(selectionOptions)) {
```

Comment:
need to run the linter

Reply:
Updated. `yarn lint` doesn't seem to be working in my dev setup.